### ISSUE TITLE
Boss egg spawning only in a region

### ISSUE THREAD

**Issue by sirben99** [kangarko/Boss#229]
Hey! Is there a way to limit the player to use the boss spawn egg only in a worldguard region? If so how do you do it? If have tried different things but cant seem to find a solution.

**Comment by kangarko** [comment]
Hey,

if you use LuckPerms, you can install ExtraContent addon (see https://www.spigotmc.org/resources/luckperms-an-advanced-permissions-plugin.28140/ and look for "WorldGuard Regions") and simply give "boss.use.spawneregg" permission in that region(s) only.

**Comment by sirben99** [comment]
> Hey,
> 
> if you use LuckPerms, you can install ExtraContent addon (see https://www.spigotmc.org/resources/luckperms-an-advanced-permissions-plugin.28140/ and look for "WorldGuard Regions") and simply give "boss.use.spawneregg" permission in that region(s) only.

Hey! I dont unfortenatly use luckyperms plugin. I have another permission plugin that i use. Sorry if im not supposed to continue this conversation you probably have better things to do but is there another way to do this or is it tiet only to a permission plugin?

**Comment by kangarko** [comment]
The plugin offers an API for developers so it is possible to code that in.

I am afraid there is no other way, what permission plugin do you use?

**Comment by sirben99** [comment]
> The plugin offers an API for developers so it is possible to code that in.
> 
> I am afraid there is no other way, what permission plugin do you use?

Hey! I use UltraPermissions plugin on my server. I also looked at the api but i dont know how to use eclipse. I also looked at your courses on coding plugins but im too broke to afford that atm. But, i did find a solution in the BOSS plugin it self and everything worked perfectly. Now the spawn egg works only in a defined worldquard region and no extra plugins/outside programs neede.

Thx for the asnwer tho!
